src/tsickle.ts

```diff
@@ -375,21 +406,39 @@ class Annotator extends ClosureRewriter {
       sourceMap: annotated.sourceMap,
     };
   }
+
+  private formatExportingDecoratorExterns() {
+    if (this.exportingDecoratorExterns.length === 0) {
+      return '';
+    } else {
+      return '\n' + this.exportingDecoratorExterns.map(name => `var ${name};\n`).join('');
```

Comment:
is `var ${name}` really the right pattern here? Shouldn't these be declared properties on an anonymous object, as we do for templates?